_pages/algorithmia-enterprise/insights.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: article
+title:  "Insights"
+excerpt: "Sending algorithm metrics to a customer-operated metrics collection system"
+categories: algorithmia-enterprise
+show_related: false
+image:
+  teaser: /icons/algo.svg
+---
+
+Insights is an algorithm metrics pipeline providing you access to your algorithm metrics payload for each algorithm session so you can measure, monitor, and manage your algorithms in production.
+
+With Insights you'll be able to opt-in to exporting your algorithm's operational metrics to external monitoring and alerting tools. This means you’ll be able to monitor your algorithm's execution time and memory consumption as well as capture any model inference metrics you expose from your algorithm's output such as predictions or accuracy. Any metrics you opt to collect will be captured and exported in a payload and, with help from a Platform Administrator, can be connected to your external systems for monitoring and alerting.
```

Comment:
Not sure if you wanted to change it for consistency, but there is a stylized apostrophe in the second `you'll`.